app/api_topologies.go

```diff
@@ -255,6 +268,30 @@ type topologyStats struct {
 	FilteredNodes      int `json:"filtered_nodes"`
 }
 
+// AddContainerFilters adds to the topology registry's containerFilters
+func AddContainerFilters(newOptions []APITopologyOption) {
+	topologyRegistry.addContainerFilters(newOptions)
+}
+
+func (r *registry) addContainerFilters(newOptions []APITopologyOption) {
+	r.Lock()
+	defer r.Unlock()
+
+	for _, o := range newOptions {
+		ContainerOpts = append(ContainerOpts, o)
```

Comment:
you can simply do `ContainerOpts = append(ContainerOpts, newOptions...)` without the loop